src/main/kotlin/me/partlysanestudios/partlysaneskies/data/skyblockdata/SkyblockItem.kt

```diff
@@ -0,0 +1,114 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+package me.partlysanestudios.partlysaneskies.data.skyblockdata
+
+class SkyblockItem(
+    val id: String,
+    val rarity: Rarity,
+    val name: String,
+    val npcSellPrice: Double,
+    var bazaarBuyPrice: Double,
+    var bazaarSellPrice: Double,
+    var averageBazaarBuy: Double,
+    var averageBazaarSell: Double,
+    var lowestBin: Double,
+    var averageLowestBin: Double,
+) {
+    companion object {
+        val emptyItem = SkyblockItem("", Rarity.UNKNOWN, "", 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
```

Comment:
I would honestly make unknown values -1.0
since there are actually items with zero sell price sometimes

Reply:
Is there?

Reply:
def some cosmetic armorstand things

Reply:
> I would differenciate between unable to sell (-1) and just no price (0)

Needs to be changed on api then

Reply:
On the hypixel API, a few things that listed as 0 coins. In game tho you cant sell them. So 0 coins means unsellable